README.md

```diff
@@ -60,6 +60,15 @@ function.k8s.io   Kubeless: Serverless framework for Kubernetes   v1
 $ kubectl get functions
 ```
 
+Expose the node port for Kubeless Controller. For example, on minikube:
+
+```console
+$ kubectl expose deployment kubeless-controller --port=8080 \
```

Comment:
thanks for the PR.

Why did you need to expose the controller deployment as NodePort ?

Reply:
I can't replicate my "fix" of exposing the NodePort, so I'll close the PR.

I see commit 3f1bd6bc569a204206b0e5fe3690ac2017395d0c to `README.md` adds a `kubectl proxy` and deletes the `minikube service get-python --url` which with the v0.1.0 yaml returns nothing (with the 0.0.18 yaml it returned `http://192.168.64.5:32115`). This fix _does_ resolve the issue.